src/Microsoft.DotNet.Darc/src/Darc/Options/VirtualMonoRepo/VmrCommandLineOptions.cs

```diff
@@ -24,17 +24,26 @@ internal abstract class VmrCommandLineOptions : CommandLineOptions
     public IServiceCollection RegisterServices()
     {
         var tmpPath = Path.GetFullPath(TmpPath ?? Path.GetTempPath());
-        var localDarcSettings = LocalSettings.LoadSettingsFile();
+        string gitHubToken = null;
+        string azureDevOpsToken = null;
+
+        try
+        {
+            var localDarcSettings = LocalSettings.LoadSettingsFile(this);
```

Comment:
You should try to put only the lines that you suspect are throwing in the `try {}`. So here it is just this line.